### ISSUE TITLE
Support for modifying src

### ISSUE THREAD

**Issue by ZKJGOLD** [goldfire/howler.js#1602]
### Feature Proposal

need to be re-instantiated each time when src for audio is changed，Can you reduce the overhead of unnecessary audio creation.

**Comment by fromage9747** [comment]
@ZKJGOLD 

I found this https://github.com/goldfire/howler.js/issues/825

I cannot use it, as _src is unknown in typescript.  Maybe you will have better luck.

I am currently unloading Howl, changing the source and reinitialising it to change a track.

**Comment by fromage9747** [comment]
Chrome on Mobile doesn't work well.  It will change the source, but it will download the entire file before it plays when the source has changed even though the player is initialised and reinitialised with `html5:true`

Initial load of the source works as expected, with `html5:true`

**Comment by fromage9747** [comment]
Further to this.  It only happens when using 3g and not on Wi-Fi.  You can check the network tab in chrome and when on 3g it downloads the entire file after changing the source, but on Wi-Fi it works as I would expect where it streams the file.

**Comment by ZenenTreadwell** [comment]
Yea it seems like a nice library but there's no way I'm picking it up until it can do something like that. Different use cases, I suppose.

**Comment by fromage9747** [comment]
@ZenenTreadwell yeah, like most people that start using this library, I moved away from it and just use the standard browser audio. This library is more suited for game that need to play many different audio files at once.
